### ISSUE TITLE
Site Admin action icons are not obvious what they do

### ISSUE THREAD

**Issue by digitaldrummerj** [HTBox/allReady#1245]
From a user experience standpoint, the icons that are on the site admin page are not obvious without having to rollover them.  Personally I prefer to have some kind of text along with the icon (if there even is an icon) so that I don't have to rollover them to figure out what they are.  

Also, if you are on a mobile device using the website, there is no way to rollover them.

Here are the icons that I am talking about.  You can view the page locally at [http://localhost:48408/Admin/Site](http://localhost:48408/Admin/Site)

![image](https://cloud.githubusercontent.com/assets/708423/18652457/7de195d6-7e87-11e6-9271-1445f736aaef.png)

**Comment by smahesh5** [comment]
Cannot open the link above. Error: "Web page does not exist"

**Comment by digitaldrummerj** [comment]
@smahesh5 that is the localhost url when you are running the site out of Visual Studio

**Comment by YannickMeeus** [comment]
I've had a stab at replacing the icons with text, but I'm not convinced as to how it makes it better. It makes it clearer but it feels it detracts from not just the feel of the menu but also from the overall feel of the page. It becomes a lot more cluttered as a result:

![admin pane - text instead of icons](https://cloud.githubusercontent.com/assets/588607/19908457/57f26a56-a07b-11e6-85fc-8e4a4c39bdfe.png)

Any opinions? I can shove this branch up to my fork if people are interested in the code changes, but it'd be cool to just get some feedback on the visuals of the page.

**Comment by digitaldrummerj** [comment]
@YannickMeeus I agree that it makes it way more obvious as to what each button does now but makes the page look a bit more cluttered.  I do like that at least they are obvious now where before you had no clue what the buttons did.  With a little bit of UX love I think you could make it work.  Might just have to rethink how the page works.

some potential ideas:

* hide the buttons by default with an action button that pops the buttons down into view
* make multiple rows of buttons
* move the buttons except edit and delete into an edit dialog.  Then clicking edit would open the edit dialog and have all of the additional buttons

**Comment by tonysurma** [comment]
From a user perspective options 1 and three would be nice

**Comment by YannickMeeus** [comment]
I'm looking at option three right now.

So we'll have "Edit" and "Delete" as primary options, and when the "Edit"button is clicked, the user is presented with the existing options and one additional one, called "Manage Skills". The main reason for this additional button being the current behaviour of the "Edit" button, which is a redirect to a page that only allows the user to add or remove skills from another user. This seems too distinct a page to be hidden behind a generic "Edit" button, and it allows me to keep the "Edit" button as an action to pop up a modal containing any of the following buttons (visibility is determined by administrator access level):

- Manage Skills
- Reset Password
- Make Site Admin
- Make Org Admin
- Revoke Site Admin
- Revoke Org Admin
- Allow API Access
- Manage Keys

Once this is done, I'll have a look at spinning off another issue to re-brand the "Edit User" page to be more specific, if that's what people want.

**Comment by tonysurma** [comment]
This sounds great and thankful for the clean up of UX which will be key to helping users onboard and use the system.

**Comment by EmilyLuijbregts** [comment]
Hi @YannickMeeus Just wanted to check in and see how you were getting on?

**Comment by YannickMeeus** [comment]
Sorry, Black Friday snuck up on me and claimed every bit of life away for a couple of weeks. I've tried to merge/rebase the 100-odd commits since my last one and it turned into a nightmare to merge, so I'll be quickly redoing my changes manually. I should have a PR ready by the end of the weekend.

**Comment by tonysurma** [comment]
Thanks @YannickMeeus

**Comment by YannickMeeus** [comment]
@tonysurma I reckon this one can be closed now 😃
